modin/pandas/utils.py

```diff
@@ -40,100 +40,26 @@
     + "https://pandas.pydata.org/pandas-docs/stable/indexing.html#attribute-access"
 )
 
-
-def from_non_pandas(df, index, columns, dtype):
-    """
-    Convert a non-pandas DataFrame into Modin DataFrame.
-
-    Parameters
-    ----------
-    df : object
-        Non-pandas DataFrame.
-    index : object
-        Index for non-pandas DataFrame.
-    columns : object
-        Columns for non-pandas DataFrame.
-    dtype : type
-        Data type to force.
-
-    Returns
-    -------
-    modin.pandas.DataFrame
-        Converted DataFrame.
-    """
-    from modin.core.execution.dispatching.factories.dispatcher import FactoryDispatcher
-
-    new_qc = FactoryDispatcher.from_non_pandas(df, index, columns, dtype)
-    if new_qc is not None:
-        from .dataframe import DataFrame
-
-        return DataFrame(query_compiler=new_qc)
-    return new_qc
-
-
-def from_pandas(df):
-    """
-    Convert a pandas DataFrame to a Modin DataFrame.
-
-    Parameters
-    ----------
-    df : pandas.DataFrame
-        The pandas DataFrame to convert.
-
-    Returns
-    -------
-    modin.pandas.DataFrame
-        A new Modin DataFrame object.
-    """
-    from modin.core.execution.dispatching.factories.dispatcher import FactoryDispatcher
-
-    from .dataframe import DataFrame
-
-    return DataFrame(query_compiler=FactoryDispatcher.from_pandas(df))
-
-
-def from_arrow(at):
-    """
-    Convert an Arrow Table to a Modin DataFrame.
-
-    Parameters
-    ----------
-    at : Arrow Table
-        The Arrow Table to convert from.
-
-    Returns
-    -------
-    DataFrame
-        A new Modin DataFrame object.
-    """
-    from modin.core.execution.dispatching.factories.dispatcher import FactoryDispatcher
-
-    from .dataframe import DataFrame
-
-    return DataFrame(query_compiler=FactoryDispatcher.from_arrow(at))
-
-
-def from_dataframe(df):
-    """
-    Convert a DataFrame implementing the dataframe exchange protocol to a Modin DataFrame.
-
-    See more about the protocol in https://data-apis.org/dataframe-protocol/latest/index.html.
-
-    Parameters
-    ----------
-    df : DataFrame
-        The DataFrame object supporting the dataframe exchange protocol.
-
-    Returns
-    -------
-    DataFrame
-        A new Modin DataFrame object.
-    """
-    from modin.core.execution.dispatching.factories.dispatcher import FactoryDispatcher
-
-    from .dataframe import DataFrame
-
-    return DataFrame(query_compiler=FactoryDispatcher.from_dataframe(df))
+from_pandas = func_from_deprecated_location(
+    "from_pandas",
+    "modin.pandas.io",
+    "Importing ``from_pandas`` from ``modin.pandas.utils`` is deprecated and will be removed in 0.28.0 release. This function was moved to ``modin.pandas.io``, please import it from there instead.",
```

Comment:
```suggestion
    "Importing ``from_pandas`` from ``modin.pandas.utils`` is deprecated and will be removed in a future version. This function was moved to ``modin.pandas.io``, please import it from there instead.",
```
change everywhere

Reply:
@dchigarev It would also be great not to make such a long line, it’s not convenient to read.